luigi_pipeline/tests/test_seqr_sv_loading.py

```diff
@@ -254,7 +254,7 @@ def setUp(self):
     def tearDown(self):
         shutil.rmtree(self._temp_dir.name)
 
-    @mock.patch('seqr_sv_loading.load_gencode', return_value=GENE_ID_MAPPING)
+    @mock.patch('luigi_pipeline.seqr_sv_loading.load_gencode', return_value=GENE_ID_MAPPING)
```

Comment:
Fixing a bug from the packaging pr... I've fixed it on another pr but it's not merged yet.